challenge.rb

```diff
@@ -1,26 +1,26 @@
 
 def capitalize_each_string(input)
-  #implement your solution here
+  input.map(&:capitalize)
 end
 
 def fetch_the_dog(input)
-  #implement your solution here
+  input.find_all { |animal| animal == 'dog' }
 end
 
 def no_dogs_allowed(input)
-  #implement your solution here
+  input.reject { |animal| animal == 'dog' }
 end
 
 def count_the_animals(input)
-  #implement your solution here
+  input.count
 end
 
 def fetch_the_first_two(input)
-  #implement your solution here
+  input.take(2)
 end
 
 def fetch_CD_animals(input)
-  #implement your solution here
+  input.select { |animal| animal == 'cat' || animal == 'dog' }
```

Comment:
Based on the name of the method and other solutions, I think the intent is to return any entries that begin with `c` or `d`.